components/nexus-httpclient/src/main/java/org/sonatype/nexus/httpclient/config/TimeoutConfigurationDeserializer.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Sonatype Nexus (TM) Open Source Version
+ * Copyright (c) 2008-2015 Sonatype, Inc.
+ * All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+ *
+ * This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+ * which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+ *
+ * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+package org.sonatype.nexus.httpclient.config;
+
+import java.io.IOException;
+
+import org.sonatype.sisu.goodies.common.Time;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.deser.std.StdDeserializer;
+
+/**
+ * {@link ConnectionConfiguration#timeout} deserializer.
+ *
+ * Considers timeout value as seconds.
+ *
+ * @since 3.0
+ */
+public class TimeoutConfigurationDeserializer
```

Comment:
I rename it to SecondsDeserialiser. It can not be TimeDeserializer because is converting seconds not any generic value.